root/components/UserAccountTabs.js

```diff
@@ -22,14 +22,14 @@ function buildTabs(
 ): $ReadOnlyArray<React.Element<'li'>> {
   const viewingOwnProfile = $c.user && $c.user.id === user.id;
   const showAdmin = $c.user && $c.user.is_account_admin;
-  const showPrivate = showAdmin || viewingOwnProfile;
+  const showPrivate = showAdmin /*:: === true */ || viewingOwnProfile;
```

Comment:
Maybe just wrap `viewingOwnProfile` and `showAdmin` in `!!( ... )` so that those are always boolean. Then you shouldn't need any of the `/*:: === true */` in this file.